base/platform/linux/base_file_utilities_linux.cpp

```diff
@@ -181,6 +181,10 @@ bool ShowInFolder(const QString &filepath) {
 }
 
 QString CurrentExecutablePath(int argc, char *argv[]) {
+#ifdef DESKTOP_APP_EXECUTABLE_PATH_OVERRIDE
+	return QString(DESKTOP_APP_EXECUTABLE_PATH_OVERRIDE);
```

Comment:
```suggestion
	return uDESKTOP_APP_EXECUTABLE_PATH_OVERRIDE_q;
```

Reply:
@ilya-fedin Thanks for the feedback, your https://github.com/desktop-app/lib_base/pull/115 looks much better, so I'll just drop the second OpenBSD related commit introducing `DESKTOP_APP_EXECUTABLE_PATH_OVERRIDE`.